.github/workflows/code_formatting.yml

```diff
@@ -39,24 +44,24 @@ jobs:
         id: black-git-check
         run: echo modified=$(if git diff-index --quiet HEAD --; then echo "false"; else echo "true"; fi) >> $GITHUB_OUTPUT
 
-      - name: Commit changes
+      - name: Push black changes
         if: steps.black-git-check.outputs.modified == 'true'
-        run: git commit -am "Fixup. Format code with Black"
+        run: |
+          git commit -am "Fixup. Format code with Black"
+          git push
```

Comment:
pourquoi avoir rajouter un git push à chaque fois ? et pas une seule fois à la fin ?

Reply:
Y'avait des erreurs "Your branch is ahead of 'origin/develop' by 1 commit. Make a push first."

Reply:
ok, étrange !